code/datums/mutations/hulk.dm

```diff
@@ -11,6 +11,10 @@
 /datum/mutation/human/hulk/on_acquiring(mob/living/carbon/human/owner)
 	if(..())
 		return
+	if(jobban_isbanned(owner, ROLE_HULK))
+		on_losing(owner)
+		to_chat(owner, "<span class='warning'>Your muscles suddenly shrink back to normal! You feel the gods looking down upon you in shame...</span>")
+		return
```

Comment:
This seems a little OOC. Can't we make up something about Centcom putting in genetic kill switches in some of their employees... genes to stop them abusing hulk? Or something?

Reply:
I've [argued against this earlier](https://github.com/tgstation/tgstation/pull/40037#discussion_r214502175). In an earlier commit, this was exactly what was said here. NT directly restricting people from hulk when hulk has to be researched and discovered by geneticists, doesn't make a 
lot of sense. I encourage you to come up wih something better than an NT cop-out.

Reply:
I don't have any better ideas, but I thought we were aiming for less OOC in IC statements like SPACEGODS ARE ASHAMED OF YOU

Reply:
"On second thought, your muscles feel just fine :)"

Reply:
Again, it might be better to not even dance around it and just say "You are banned from being a hulk."

Reply:
I'm leaving it. The other in game role bans are not icly subtle not are they completely oocly blunt. If the maintainers have a huge problem with it then we'll talk, but it has very little impact on anything itself.